toolbox

```diff
@@ -765,6 +807,9 @@ usage()
     echo "               enter [--container <name>]"
     echo "                     [--release <release>]"
     echo "   or: toolbox [-v | --verbose]"
+    echo "               run [-c | --container <name>]"
```

Comment:
Nitpick: I'd put it after `rm` to keep them sorted.

Reply:
no problem, will fix that.